shoes-core/spec/shoes/shared_examples/state.rb

```diff
@@ -13,6 +13,7 @@
   end
 
   it "should disable" do
+    subject.state = ""
```

Comment:
Might be, but I just needed it to have a value change from initialization (which is setting "disabled") since style skips no-op sets.